handler/implementations/sysDevicesVirtual.go

```diff
@@ -40,6 +41,7 @@ import (
 
 type SysDevicesVirtual struct {
 	domain.HandlerBase
+	passthruNodes map[string]bool
```

Comment:
Instead of creating a new separate map, I would place this property (`passthruNode bool`) within the existing EmuResourceMap so that we can easily re-use this approach in other handlers in the future.

Reply:
I am not sure: in this case we are not emulating the `/sys/virtual/devices/net` resource, so it doesn't belong in the `EmuResourceMap`. Thus it would be confusing to add it there.

Reply:
Got your point. LGTM.